### PR TITLE
[MH2018-30] Feature/sh pagination controls

### PR DESCRIPTION
**Pull request by jaimelr**
I created the pagination controls to navigate trough products in a specific collection.
Flat:
paginate-controls

The development store:
paginate-control-real

https://fiorella-dev-2.myshopify.com

### CODE REVIEW

File: fiorella/src/styles/layout/_pagination-controls.scss [hunk 1]
```diff
@@ -0,0 +1,26 @@
+.paginate-control {
+  display: flex;
```
Comment (magmahound) [inline fiorella/src/styles/layout/_pagination-controls.scss]: Properties should be ordered align-items, display

File: fiorella/src/snippets/pagination.liquid [hunk 2]
```diff
@@ -1,12 +1,32 @@
-<div class="pagination">
+<div class="pagination pagination-controls">
   {% if paginate.previous.is_link %}
-    <a href="{{ paginate.previous.url }}" class="prev-button">« PREV</a>
+  <a href="{{ paginate.previous.url }}" class="prev-button">
```
Comment (8geonirt) [inline fiorella/src/snippets/pagination.liquid]: This code can be simplified by creating a variable using liquid, one for the `url` that is going to be inside the `href` of the anchor, and another one for the `class` of the anchor as well... it's just a suggestion

Comment (8geonirt) [reply fiorella/src/snippets/pagination.liquid]: Actually, this code can be simplified to be just a small piece of HTML... as I said, it depends on you if you keep as it is right now, if you want to simplify the code then go ahead 🤓

File: fiorella/src/styles/pages/_collection.scss [hunk 3]
```diff
@@ -93,6 +93,12 @@
     text-decoration: none;
   }
 
+  .paginate-control {
+    p {
```
Comment (magmahound) [inline fiorella/src/styles/pages/_collection.scss]: Selector should have depth of applicability no greater than 2, but was 3

**Pull request merged**